### DISCUSSION TITLE
part-ii/2017-73/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#83] [Question comments]
Part II 2017 General Relativity: Paper 3, Section II, D 

https://questions.tripos.org/part-ii/2017-73/

**Comment by AndoBando** [comment]
Every null curve in $\mathcal{M}$ is geodesic, but not an affinely parametrized one.
The geodesics of Minkowksi spacetime are just straight lines.
